src/project/qml/MuseScore/Project/internal/Export/PdfSettingsPage.qml

```diff
@@ -55,4 +55,25 @@ ExportSettingsPage {
             }
         }
     }
+
+    // Add transparency option
+    CheckBox {
+        width: parent.width
+        text: qsTrc("project/export", "Transparent background")
+
+        navigation.name: "TransparentBackgroundCheckbox"
+        navigation.panel: root.navigationPanel
+        navigation.row: root.navigationOrder + 2
+
+        checked: root.model.pdfTransparentBackground
+
+        onClicked: {
+            root.model.pdfTransparentBackground = !checked
+        }
+    }
+
+    StyledTextLabel {
+        width: parent.width
+        text: qsTrc("project/export", "Each page of the selected parts will be exported as a separate %1 file.").arg("PDF")
+    }
```

Comment:
I believe this label shouldn't be here; the text is not accurate, and for PDF there are already those radio buttons to select how parts should be exported